src/lib/config.ts

```diff
@@ -665,11 +665,6 @@ export class default_config {
      */
     viewsource: "tridactyl" | "default" = "tridactyl"
 
-    /**
-     * Which storage to use. Sync storage will synchronise your settings via your Firefox Account.
-     */
-    storageloc: "sync" | "local" = "sync"
```

Comment:
I think `:set` should emit a warning when trying to set storageloc. Some users might do that in their tridactylrc

Reply:
I had chosen not to do this on purpose as the only reason to use it would be `set storageloc local`, which would be the same behaviour as this PR.

I could be persuaded to have `set storageloc sync` throw an error but I suspect no-one uses that.

Happy to be persuaded otherwise :)

Reply:
https://github.com/search?q=%22storageloc+sync%22&type=Code :)